third_party/gpus/cuda/BUILD.tpl

```diff
@@ -58,10 +58,8 @@ cc_library(
     linkopts = [
         "-ldl",
         "-lpthread",
-    ] + select({
-        "@//tensorflow:darwin": [],
```

Comment:
It looks like we still need this change.

This `BUILD` file is generated in the `@local_config_cuda` workspace, which is created by `cuda_configure`. The `@//` syntax refers to a label in the main workspace, which is fine when building from the `tensorflow` workspace since `//tensorflow:darwin` exists. 

Changing this from `@//` to `//` breaks the build because then the `//tensorflow:darwin` resolves to `@local_config_cuda//tensorflow:darwin` does not exist.

Similarly, when building out of the `serving` workspace, this also breaks because `//tensorflow:darwin` does not exist in `serving` (it would be `//tensorflow/tensorflow:darwin` based on the way the repository is set up currently).

In any case, since `cuda_configure` already performs platform-specific checks, it would be more consistent to keep this change which moves the platform-specific check into `cuda_configure`.